addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -415,7 +419,12 @@ var KanbanRenderer = BasicRenderer.extend({
 
         // append ghost divs to ensure that all kanban records are left aligned
         var prom = Promise.all(self.defs).then(function () {
-            return self._renderGhostDivs(fragment, 6);
+            var options = {};
+            var $recordView = $(fragment).find(".o_kanban_record:not(.o_kanban_ghost)");
+            if ($recordView.length && $recordView.attr("style")) {
```

Comment:
Above in this function, you have `var kanbanRecord = new KanbanRecord ...`. Define the variable outside of the foreach so that the last kanban record is still accessible here.

Then here you can do:
```javascript
if (kanbanRecord) {
    var style = kanbanRecord.$el.attr('style');
    if (style) {
        options.inlineStyle = style;
    }
}
```
See:
- No need of new search in the DOM
- No duplicate call to `attr`
- Also use `'` instead of `"` for technical strings (convention)